control.extensions/src/test/java/Hyperlink.java

```diff
@@ -0,0 +1,20 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import ControlExtensions.Angular.ControlExtension;
+
+public class Hyperlink extends ControlExtension {
+
+	public Hyperlink(WebElement mappedElement) {
+		super(mappedElement);
+	}
+	
+	public void click() {
+		mappedElement.click();
+	}
+	
+	public String getResponse() {
+		return mappedElement.findElement(By.xpath("//p[@id='linkResponse']")).getText();
```

Comment:
// will search from the root of the DOM. If you intend to search from the mappedElement then use ./